fem/eltrans.cpp

```diff
@@ -444,6 +444,8 @@ int IsoparametricTransformation::OrderJ() const
          return (FElem->GetOrder()-1);
       case FunctionSpace::Qk:
          return (FElem->GetOrder());
+      case FunctionSpace::Uk:
+         return (FElem->GetOrder()); // Experimental
```

Comment:
The "experimental" comment may no longer be appropriate here and for other cases of `FunctionSpace::Uk` below